dev/source/docs/ardupilot-on-linux-init-scripts.rst

```diff
@@ -0,0 +1,38 @@
+.. _ardupilot-on-linux-init-scripts:
+
+============================
+Linux Initialization Scripts
+============================
+
+
+Example Systemd Unit File
+=========================
+
+Adjust the following and place in /etc/systemd/system/arduplane.service
+
+::
+
+    [Unit]
+    Description=ArduPlane Service
+    Conflicts=arducopter.service ardupilot.service ardurover.service
+
+    [Service]
+    EnvironmentFile=/etc/default/arduplane
+
+    # Lock CPU governor to top speed for reduced latency
+    ExecStartPre=/bin/bash -c "/bin/echo performance | tee /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor"
+
+    # Steer any IRQs Ex: IRQ#30 to CPU 3
+    ExecStartPre=/bin/bash -c "echo 3 > /proc/irq/30/smp_affinity_list"
+
+    # arduplane with selected serial port options, which must be configured per board.
+    ExecStart=/usr/local/bin/arduplane -serial0 /dev/ttyAMA0 -serial1 /dev/ttyS0
+
+    Restart=on-failure
+
+To start, ``sudo systemctl start arduplane``.
+To check on it, ``sudo systemctl status arduplane``.
+To restart, ``sudo systemctl restart arduplane``.
+To stop, ``sudo systemctl restart arduplane``.
```

Comment:
--> To stop, ``sudo systemctl stop arduplane``.

Reply:
@rsaxvc do not mark as resolved unless you have pushed a resolution